tldr/hybrid_extractor.py

```
@@ -3151,6 +3168,159 @@ def _extract_elixir_function(self, node, source: bytes, is_private: bool) -> Fun
+            elif child.type == "parameters":
+                for param in child.children:
+                    if param.type == "parameter":
+                        param_name = None
+                        for pc in param.children:
+                            if pc.type == "identifier":
+                                param_name = self._safe_decode(source[pc.start_byte:pc.end_byte])
+                                break
+                        if param_name:
+                            params.append(param_name)
+            elif child.type == "pub":
+                is_public = True
```

Ruff (0.14.14)

Local variable `is_public` is assigned to but never used

Remove assignment to unused variable `is_public`

(F841)